src/SprykerMiddleware/Zed/Process/Dependency/Plugin/Configuration/ProcessConfigurationPluginInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Dependency\Plugin\Configuration;
+
+use SprykerMiddleware\Zed\Process\Dependency\Plugin\Iterator\ProcessIteratorPluginInterface;
+use SprykerMiddleware\Zed\Process\Dependency\Plugin\Log\MiddlewareLoggerConfigPluginInterface;
+
+interface ProcessConfigurationPluginInterface
+{
+    /**
+     * @return string
+     */
+    public function getProcessName(): string;
+
+    /**
+     * @return \SprykerMiddleware\Zed\Process\Dependency\Plugin\Iterator\ProcessIteratorPluginInterface
+     */
+    public function getIteratorPlugin(): ProcessIteratorPluginInterface;
+
+    /**
+     * @return \SprykerMiddleware\Zed\Process\Dependency\Plugin\StagePluginInterface[]
+     */
+    public function getStagePlugins(): array;
+
+    /**
+     * @return \SprykerMiddleware\Zed\Process\Dependency\Plugin\Log\MiddlewareLoggerConfigPluginInterface
+     */
+    public function getLoggerPlugin(): MiddlewareLoggerConfigPluginInterface;
```

Comment:
Can we provide absract class with implementation of this method which would make use of getDefaultLoggerConfigPlugin() method? It's not that convenient if each process configuration would call the same method